templates/include/metadata/ingestionCommonMetadata.1.schema.json

```diff
@@ -93,4 +93,18 @@
       "type": "string"
     }
   }
+},
+"proxy": {
+  "type": "object",
+  "description": "Results of Proxy detection based on X-Forwarded-For",
+  "properties": {
+    "list_versions": {
+      "description": "The specific proxy subnet list versions used for this detection",
```

Comment:
CSV?

Reply:
right now it's semicolon separated, <source>:<timestamp>, but I'm not attached to that format, it was just the first thing that came to mind when implementing:
https://github.com/mozilla/gcp-ingestion/blob/7780f9e073a16bbf9d43cf47bcda33a6e2585551/ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParseProxyTest.java#L77-L79